project/DockerPublish.scala

```diff
@@ -1,60 +1,29 @@
 import com.typesafe.sbt.packager.Keys.*
 import com.typesafe.sbt.packager.docker.Cmd
-import com.typesafe.sbt.packager.docker.DockerPlugin.autoImport.Docker
-import sbt.Keys.*
+import com.typesafe.sbt.packager.docker.DockerPlugin.autoImport.{dockerBuildxPlatforms, Docker}
 import sbt.*
 
-import scala.sys.process.Process
-
 object DockerPublish {
 
-  lazy val dockerSettings = imageSettings ++ dockerBuildxSettings
-
-  lazy val ensureDockerBuildx    = taskKey[Unit]("Ensure that docker buildx configuration exists")
-  lazy val dockerBuildWithBuildx = taskKey[Unit]("Build docker images using buildx")
+  lazy val dockerSettings: Seq[Def.Setting[?]] = imageSettings
 
   private lazy val imageSettings = Seq(
-    Docker / packageName := "kafka-message-scheduler",
-    dockerBaseImage      := "alpine:3.21",
-    dockerRepository     := registry,
-    dockerLabels         := Map("maintainer" -> "Sky"),
-    dockerUpdateLatest   := true,
+    Docker / packageName  := "kafka-message-scheduler",
+    dockerBaseImage       := "alpine:3.21",
+    dockerRepository      := Some(registry),
+    dockerLabels          := Map("maintainer" -> "Sky"),
+    dockerUpdateLatest    := true,
+    dockerBuildxPlatforms := Seq("linux/arm64", "linux/amd64"),
     dockerCommands ++= Seq(
       Cmd("USER", "root"),
       Cmd("RUN", "apk add --no-cache bash openjdk17")
     ),
     dockerAliases ++= additionalRegistries.map(host => dockerAlias.value.withRegistryHost(Some(host)))
   )
 
-  val allRegistries                     = sys.env.get("CONTAINER_REPOSITORIES").fold(List("test"))(_.split(" ").toList)
-  val registry                          = allRegistries.headOption // Provide a docker registry host
-  val additionalRegistries              = allRegistries.drop(1)    // Remove the first host, because it is already provide.
-  private lazy val dockerBuildxSettings = Seq(
```

Comment:
Support for buildx in sbt-native-packager was added in [sbt/sbt-native-packager@`0b8e83a` (#1527)](https://github.com/sbt/sbt-native-packager/pull/1527/commits/0b8e83a8b418510ced9c1bac152d8cf977d9ae88) so this is not needed